### DISCUSSION TITLE
How do I use IIS proper as a debug target?

### DISCUSSION THREAD

**Discussion by catmanjan** [dotnet/aspnetcore#57073] [Q&A]
In old dotnet I could debug a project and it would run in IIS proper (not IIS express)

Now the only debug profiles I get are Kestrel and IIS Express

How can I target an actual IIS site?

**Comment by gfoidl** [comment]
Does [Debug ASP.NET or ASP.NET Core apps in Visual Studio](https://learn.microsoft.com/en-us/visualstudio/debugger/how-to-enable-debugging-for-aspnet-applications?view=vs-2022) provide the information needed?

**Comment by catmanjan** [comment]
I get stuck at step 3: For local IIS, select New and create a new IIS
profile.

I don't get an option to select IIS profile, I only get Kestrel and IIS
express
